substrate-processor/src/interfaces/substrate.ts

```diff
@@ -159,6 +159,25 @@ export interface EvmLogEvent extends SubstrateApplyExtrinsicEvent {
 }
 
 
+export interface EvmExecutedLog {
+    address: string
+    data: string
+    topics: string[]
+}
+
+
+export interface EvmExecutedEvent extends SubstrateApplyExtrinsicEvent {
+    name: 'EVM.Executed'
+    args: {
```

Comment:
I'm a bit worried about typing `args` without checks. May be do it like `.typed.asXXX`?

@dzhelezov

Reply:
I didn't think about it and probably you are right!

Reply:
Or, maybe let's just type it as `any` for now.